src/main/kotlin/org/wfanet/measurement/gcloud/spanner/Mutations.kt

```diff
@@ -0,0 +1,118 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.wfanet.measurement.gcloud.spanner
+
+import com.google.cloud.ByteArray
+import com.google.cloud.Timestamp
+import com.google.cloud.spanner.Mutation
+import com.google.protobuf.Message
+import com.google.protobuf.ProtocolMessageEnum
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setBoolean")
+fun Mutation.WriteBuilder.set(columnValuePair: Pair<String, Boolean>): Mutation.WriteBuilder {
+  val (columnName, value) = columnValuePair
+  return set(columnName).to(value)
+}
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setBooleanNullable")
+fun Mutation.WriteBuilder.set(columnValuePair: Pair<String, Boolean?>): Mutation.WriteBuilder {
+  val (columnName, value) = columnValuePair
+  return set(columnName).to(value)
+}
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setLong")
+fun Mutation.WriteBuilder.set(columnValuePair: Pair<String, Long>): Mutation.WriteBuilder {
+  val (columnName, value) = columnValuePair
+  return set(columnName).to(value)
+}
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setLongNullable")
+fun Mutation.WriteBuilder.set(columnValuePair: Pair<String, Long?>): Mutation.WriteBuilder {
+  val (columnName, value) = columnValuePair
+  return set(columnName).to(value)
+}
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setDouble")
+fun Mutation.WriteBuilder.set(columnValuePair: Pair<String, Double>): Mutation.WriteBuilder {
+  val (columnName, value) = columnValuePair
+  return set(columnName).to(value)
+}
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setDoubleNullable")
+fun Mutation.WriteBuilder.set(columnValuePair: Pair<String, Double?>): Mutation.WriteBuilder {
+  val (columnName, value) = columnValuePair
+  return set(columnName).to(value)
+}
+
+/** Sets the value that should be bound to the specified column. */
+@JvmName("setString")
```

Comment:
why not setStringNullable?

Reply:
We don't need to have distinct nullable and non-nullable overloads for String since it's not a JVM boxed type. The other methods that end in "Nullable" only exist because there's a separate non-nullable version which avoids autoboxing.

Reply:
Switched to "Boxed" instead of "Nullable" to clarify.